tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py27,py36,py37,quality
+
+[testenv]
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/dev-requirements.txt
+
+whitelist_externals =
+    env
+
+environment =
+    CODEJAIL_TEST_USER=sandbox
+
+commands =
+    virtualenv {envtmpdir}/venv
```

Comment:
Creating a new virtualenv inside another virtualenv seems like it could come back to bite us later, but I can't think of a showstopping problem with it off the top of my head.